app/src/main/java/org/liberty/android/fantastischmemo/ui/QuizActivity.java

```diff
@@ -352,64 +414,88 @@ private void setupGradeButtons() {
     private CharSequence getActivityTitleString() {
         StringBuilder sb = new StringBuilder();
         sb.append(getString(R.string.quiz_text) + ": " + (totalQuizSize - queueManager.getNewQueueSize()) + "/" + totalQuizSize + " ");
-        sb.append(getString(R.string.review_short_text) + ": " + queueManager.getReviewQueueSize()+ " ");
+        sb.append(getString(R.string.review_short_text) + ": " + queueManager.getReviewQueueSize() + " ");
         sb.append(getString(R.string.id_text) + ": " + getCurrentCard().getId() + " ");
         if (!Strings.isNullOrEmpty(getCurrentCard().getCategory().getName())) {
             sb.append(getString(R.string.category_short_text) + ": " + getCurrentCard().getCategory().getName());
         }
         return sb.toString();
     }
 
+    private String timeToCompleteQuiz() {
+        String timeToCompleteText = "";
+        long timeToCompleteInMilliseconds = (timeInSeconds * 1000) - timeLeftInMilliseconds;
+        int timeInMinutes = (int) timeToCompleteInMilliseconds / 60000;
+        int timeInSeconds = (int) timeToCompleteInMilliseconds % 60000 /1000;
```

Comment:
Why did you put % 60000/1000 and not just % 60?

Reply:
because im using the variable timeToCompleteInMilliseconds which is in ms